test/clj_test_containers/core_test.clj

```diff
@@ -5,7 +5,7 @@
 (deftest init-test
   (testing "Testing basic testcontainer generic image initialisation"
     (let [container (create {:image-name "postgres:12.2"
-                             :exposed-ports [5432] 
```

Comment:
Note this is the example of fix done by running `lein cljfmt fix`.

Reply:
Thanks again for your suggestion!
I had bad experiences with autoformatters, also as far as I understand cljfmt does not change line endings, but only identations? Line endings would still be OS dependent.

I will add a proper `.gitattributes` file in the next days, to make sure git converts all line breaks to a unique format on checkin. I should have added it to the repository in the beginning, anyways. This would fix the line break issue for everybody.

Reply:
I have added the configuration to .gitattributes, now. If you pull the latest version and run the following command, everything should be harmonized automatically by git:

`git add --renormalize .`

I would suggest excluding the autoformatter from the Pull Request, again, if this works? Then we can merge your contributions into master 🙂

Reply:
I reverted my changes that remove auto-formatter from the code base.
And ran the command you mentioned.
Please review.
Thanks